datafusion/sqllogictest/test_files/errors.slt

```diff
@@ -161,3 +161,12 @@ create table records (timestamp timestamp, value float) as values (
     '2021-01-01 00:00:00', 1.0,
     '2021-01-01 00:00:00', 2.0
 );
+
+statement ok
+CREATE TABLE tab0(col0 INTEGER, col1 INTEGER, col2 INTEGER);
+
+statement ok
+INSERT INTO tab0 VALUES(83,0,38);
+
+query error DataFusion error: Arrow error: Divide by zero error
+SELECT DISTINCT - 84 FROM tab0 AS cor0 WHERE NOT + 96 / + col1 <= NULL GROUP BY col1, col0;
```

Comment:
- AS-IS : `query error DataFusion error: External error: External error: External error: Arrow error: Divide by zero error`
- An ExternalError is added to each RepartitionExec.

<img width="861" alt="스크린샷 2025-02-02 오전 12 05 06" src="https://github.com/user-attachments/assets/24541370-e6c1-4e3e-b218-ef27f395efd3" />